spec/granite/transactions/save_spec.cr

```diff
@@ -15,12 +15,6 @@ describe "#save" do
     parent.persisted?.should be_false
   end
 
-  pending "does not save a model with type conversion errors" do
-    model = Comment.new(articleid: "foo")
-    model.errors.size.should eq 1
-    model.save.should be_false
-  end
```

Comment:
This wouldn't compile anymore since the initializer would be expecting an `Int64?` not `String`.  If the user wanted they could accept both and set `articleid` within the initialize method; converting the input value if needed.